### PR TITLE
move log in button to proper place, wrap login form in keyboard avoiding view

### PR DESCRIPTION
**Pull request by osdiab**
Closes #260.

# Android

## Log in form

https://send.firefox.com/share/e58a631e84

## Give form

https://send.firefox.com/download/8c1ba43d60/#on8kSFl_Q9gdHMfFiep-Mg

# iOS

### Accessibility large text
keyboard-avoiding-accessibility

### iPhone 5s small screen
keyboard-avoiding-iphone-5

### Accessibility large text
keyboard-avoiding-give-accessibility

### iPhone 5s small screen
keyboard-avoiding-give-iphone-5

### CODE REVIEW

**Comment by tinaroh** [comment]
If possible, can you also test on Android too? Keyboard avoiding form wasn't working properly for me for Give page and I had to wrap things inside a ScrollView.

**Comment by osdiab** [comment]
@tinaroh I just reimplemented it altogether removing the scroll view. I don't have an android device to test with, but I'll ask Rahul to/maybe just suck it up and buy a cheap android phone somewhere lol

**Comment by osdiab** [comment]
@rahulgi care to help with that?

**Comment by rahulgi** [comment]
Unfortunately does not seem to keep the buttons visible on my phone. What if we put the buttons side by side instead of stacked?

screenshot_20180902-150930

File: src/components/pages/LogIn.tsx [hunk 1]
```diff
@@ -213,6 +209,17 @@ class PhoneNumberForm extends React.Component<
             Google Play Services.
           </Text>
         )}
+        <Button
+          title={
+            this.props.waitingForCode
+              ? "Requesting..."
+              : phoneValid
+                ? "Request SMS Code"
+                : "Log In With Phone:"
```
Comment (rahulgi) [inline src/components/pages/LogIn.tsx]: There's an extra ":" at the end of this text.

**Comment by osdiab** [comment]
We should probably put more effort into robustly figuring out how to use KeyboardAvoidingView's vertical offset property/maybe wrap pages in `ScrollView` elements to ensure that tiny screens are still usable, but I'll save that for another PR.

**Pull request merged**